src/overlays/actors/ovl_En_Md/z_en_md.c

```diff
@@ -339,7 +339,8 @@ u16 EnMd_GetTextKokiriForest(GlobalContext* globalCtx, EnMd* this) {
         return 0x1034;
     }
 
-    if ((CUR_EQUIP_VALUE(EQUIP_SHIELD) == 1) && (CUR_EQUIP_VALUE(EQUIP_SWORD) == 1)) {
+    if ((CUR_EQUIP_VALUE(EQUIP_SHIELD) == PLAYER_SHIELD_DEKU) &&
+        (CUR_EQUIP_VALUE(EQUIP_SWORD) == PLAYER_SWORD_KOKIRI)) {
```

Comment:
This isn't quite right since player enums don't necessarily correspond to equip values and it's not what they are intended for. We don't have enums for equip values right now, tho maybe we should add them, but that's a different story and would probably better kept for a separate set of changes.